src/client/entity-editor/alias-editor/alias-editor.js

```diff
@@ -80,10 +80,10 @@ const AliasEditor = ({
 				</div>
 				<div>
 					{
-						aliases.map((alias, rowId) => (
+						aliases.map((alias) => (
 							<AliasRow
-								index={rowId}
-								key={rowId}
+								index={alias.id}
```

Comment:
Keeping `index={rowId}` here is essential for the mechanism that update the appropriate alias. See how it is used here:
https://github.com/bookbrainz/bookbrainz-site/blob/4743abfaa5845f48cf26dcd5b631bdf45140ded3/src/client/entity-editor/alias-editor/alias-row.js#L153-L176
You'll need to put that `index` back as it was. The `key={alias.id}` can stay (so that eslint does not complain)

Right now, when I edit or create an entity clicking on "Add alias" throws all kinds of errors and breaks the page.
I insist again that you *need* to try using the code that you are modifying, for each and every change. You would have caught that error.